Gems/PhysX/Code/Source/EditorHeightfieldColliderComponent.cpp

```diff
@@ -338,10 +347,57 @@ namespace PhysX
     }
 
     // DisplayCallback
-    void EditorHeightfieldColliderComponent::Display(AzFramework::DebugDisplayRequests& debugDisplay) const
+    void EditorHeightfieldColliderComponent::Display(const AzFramework::ViewportInfo& viewportInfo,
+        AzFramework::DebugDisplayRequests& debugDisplay) const
     {
-        const auto& heightfieldConfig = static_cast<const Physics::HeightfieldShapeConfiguration&>(*m_shapeConfig);
-        m_colliderDebugDraw.DrawHeightfield(debugDisplay, m_colliderConfig, heightfieldConfig);
+        AzPhysics::SimulatedBody* simulatedBody = GetSimulatedBody();
+        if (!simulatedBody)
+        {
+            return;
+        }
+
+        AzPhysics::StaticRigidBody* staticRigidBody = azrtti_cast<AzPhysics::StaticRigidBody*>(simulatedBody);
+        if (!staticRigidBody)
+        {
+            return;
+        }
+
+        // Calculate the center of a box in front of the camera - this will be the area to draw
+        const AzFramework::CameraState cameraState = AzToolsFramework::GetCameraState(viewportInfo.m_viewportId);
+        const AZ::Vector3 boundsAabbCenter = cameraState.m_position + cameraState.m_forward * physx_heightfieldDebugDrawDistance * 0.5f;
```

Comment:
suggestion: Would be great to add a AZ_CVAR that controls the whether to draw the AABB or not for debugging purposes, this could be very helpful to check it's doing what we expect

Reply:
(if it's already drawing by default that's fine, just wanted to double check)

Screenshot in the PR description can help with this sort of thing too